implementations/rust/ockam/ockam_command/src/arguments.rs

```diff
@@ -0,0 +1,44 @@
+/// Return true if the list of arguments contains a help flag
+pub fn has_help_flag(input: &[String]) -> bool {
+    !input.contains(&"-h".to_string()) && !input.contains(&"--help".to_string())
```

Comment:
Seeing how this function is used, I think the name of the function is the opposite of what it does right?

Reply:
Totally right!